src/UX/Identification/Core.php

```diff
@@ -0,0 +1,32 @@
+<?php namespace ILIAS\UX\Identification;
+
+/**
+ * Class DynamicProvider
```

Comment:
Should be core. What does core stand for? Explain here a little in the classes comment. What else will get the ids out of this other than main menu? E.g. Footer?